components/Testimonial/index.vue

```diff
@@ -13,21 +13,22 @@
           Testimonial
         </h3>
         <h4 class="testimonial__content-subtitle">
-          Dari Para Pengguna Desa Digital
+          Kata Mereka Tentang Desa Digital
         </h4>
         <div class="testimonial__content-box-desc">
           <p class="testimonial__content-description">
-            Lorem ipsum dolor sit amet,
-            consectetur adipiscing elit. Enim aenean justo, adipiscing in lacinia mauris mauris vel.
+            Manfaat Desa Digital telah dirasakan oleh berbagai pihak.
```

Comment:
Hello kak @adzharamrullah, I see on the preview page, this section doesn't have the same baseline. I think I know where the problem is kak, on class `testimonial__content-box-desc` style try to change `items-end` into `items-baseline`
![baseline](https://user-images.githubusercontent.com/79241754/144164663-a43ca524-09a6-4832-9b4f-8f70db4444e2.png)

Reply:
i think is the same result, I don't think it's a problem and it's in accordance with the design

Reply:
done fixed in https://github.com/jabardigitalservice/dedi-landing-page/pull/43/commits/c7c6d581040e8e9699fa30fca47927fa0d2be693 @doohanas